src/libraries/System.Net.Http/src/System/Net/Http/Headers/EntityTagHeaderValue.cs

```diff
@@ -23,18 +23,11 @@ public bool IsWeak
             get { return _isWeak; }
         }
 
-        public static EntityTagHeaderValue Any
+        public static EntityTagHeaderValue Any => s_any ??= new EntityTagHeaderValue();
```

Comment:
This is the only static in the whole type and the type is beforefieldinit. The manual lazy initialization is not needed. Standard inline initialization would work just fine.

Reply:
I think it was done this way so that an EntityTagHeaderValue is only created if the public Any is used, e.g. Clone below accesses the backing field directly:
https://github.com/dotnet/runtime/pull/44802/files#diff-a6d743f24358ded728fe4adec74a5810d74e0841577a5fb672f8e442220a1d26R182-R185

Reply:
(That said, from looking at the parser for entity tags, it seems likely that if an entity tag header arrives, we'll access Any.)